src/Swarm/Web.hs

```diff
@@ -51,14 +58,15 @@ import Network.Wai.Handler.Warp qualified as Warp
 import Servant
 import Servant.Docs (ToCapture)
 import Servant.Docs qualified as SD
+import Servant.Docs.Internal qualified as SD (renderCurlBasePath)
```

Comment:
This should not be a problem as the record field is reexported normally. I made a PR to eventually fix that:
* haskell-servant/servant#1706